lib/ts/utils.ts

```diff
@@ -112,6 +113,10 @@ export function validateAndNormaliseInputOrThrowError(options: InputType): Norma
         onHandleEvent = options.onHandleEvent;
     }
 
+    if (options.enableDebugLogs !== undefined && options.enableDebugLogs) {
+        enableLogging();
+    }
```

Comment:
this should go after the function call to `validateAndNormaliseInputOrThrowError` has been called.. the name of this function does not indicate any side effects, and yet, here you have introduced one.

Reply:
Updated, saw similar issue in `supertokens-website` as well, fixed it there as well, with this PR: https://github.com/supertokens/supertokens-website/pull/239